src/compiler.jl

```diff
@@ -7753,9 +7770,11 @@ end
             handleCustom("enz_noop", [StringAttribute("enzyme_inactive"; ctx), EnumAttribute("readonly"; ctx)])
             continue
         end
-        if EnzymeRules.is_inactive_from_sig(mi.specTypes; world)
-            handleCustom("enz_noop", [StringAttribute("enzyme_inactive"; ctx), StringAttribute("nofree"; ctx)])
-            continue
+        @static if EnzymeRules.issupported()
```

Comment:
Unfortunately, per the 1.6 test which we have disabled custom rules, we still need the activity checks to be available.